tools/workspace-plugin/src/generators/cypress-component-configuration/index.ts

```diff
@@ -22,7 +22,8 @@ export default async function (tree: Tree, schema: CypressComponentConfiguration
     return;
   }
 
-  addFiles(tree, normalizedOptions);
+  // @ts-expect-error TODO: remove offsetFromRoot after migration to ts 5.5
```

Comment:
why is this needed when you updated the `Options` type ?

Reply:
forgot about this type 🙈  but there was a typo and placed inside ReturnType, fixed